diff --git a/CMakeLists.txt b/CMakeLists.txt
index 559a1522dc..41718f5f23 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1439,8 +1439,10 @@ if(ENABLE_PLUGINS)
 	set(HAVE_PLUGINS 1)
 	add_custom_target(plugins)
 	set_target_properties(plugins PROPERTIES FOLDER "Plugins")
+	link_directories(build/run)
 	set(PLUGIN_SRC_DIRS
 		plugins/epan/ethercat
+		plugins/epan/tezos
 		plugins/epan/gryphon
 		plugins/epan/irda
 		plugins/epan/mate
